<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLivreWidget</class>
 <widget class="QWidget" name="QLivreWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>224</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>469</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>30000</width>
    <height>30000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QLivreWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background-color: white;
color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Titre : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LblTitre">
       <property name="text">
        <string>Harry Potter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Auteur : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LblAuteur">
       <property name="text">
        <string>JK Rolling</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Disponibilité :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LblDispo">
       <property name="styleSheet">
        <string notr="true">QLabel{
color : green
}</string>
       </property>
       <property name="text">
        <string>Libre</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Référence :  #</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LblRef">
       <property name="text">
        <string>137</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Résumé :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnVoir">
       <property name="text">
        <string>Voir</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="LblResume">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>697</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Harry, un jeunr homme de 17 ans a des pouvois magiques</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>BtnVoir</sender>
   <signal>toggled(bool)</signal>
   <receiver>QLivreWidget</receiver>
   <slot>MontrerResume(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MontrerResume(bool)</slot>
 </slots>
</ui>
